### PR TITLE
[Trivial] Enable websocket coroutines

### PR DESCRIPTION
**Pull request by miguelportilla**
David's fix to the websocket coroutines.

@JoelKatz @vinniefalco

### CODE REVIEW

**Comment by vinniefalco** [comment]
This again? I thought we just turned them off!

**Comment by miguelportilla** [comment]
@vinniefalco David found a real fix.

**Comment by JoelKatz** [comment]
This may well be the "last bug" :)
It was in the way the legacy path finding requests were created.

File: src/ripple/rpc/handlers/RipplePathFind.cpp [hunk 1]
```diff
@@ -82,8 +82,7 @@ Json::Value doRipplePathFind (RPC::Context& context)
 
     Json::Value jvResult;
 
-    if (true || // TODO MPORTILLA temp fix to disable broken websocket coroutines
-        getConfig().RUN_STANDALONE ||
+    if (getConfig().RUN_STANDALONE ||
```
Comment (vinniefalco) [inline src/ripple/rpc/handlers/RipplePathFind.cpp]: Needs to read `context.app.config()` instead of `getConfig()`

**Comment by vinniefalco** [comment]
:+1:

**Comment by vinniefalco** [comment]
Needs a rebase

**Pull request closed without merging**